### ISSUE TITLE
Automatically setting library path for VSCode-R extension

### ISSUE THREAD

**Issue by MiguelRodo** [MiguelRodo/CompTemplate#14]
You should do this each time the remote starts up, as the exact path will depend on the machine.

So, not just a postCreateCommand but an initialisation command.
Here is the path to the remote settings on GitPod:

![image](https://github.com/MiguelRodo/CompTemplate/assets/23501332/5230d1a4-9482-4834-a9d5-e16ca350baef)

Here is the path to the remote settings in a Codespace:

![image](https://github.com/MiguelRodo/CompTemplate/assets/23501332/fd554347-6ade-4e62-b6ea-3d76196124bc)

So, basically, it always has `.vscode-remote/data/Machine/settings.json`, but on VS Code this is located within `/home/code-space/` whereas on GitPod it is within `/workspace/`. 

So, for a `.devcontainer` you use the `postStartCommand` key ([ref](https://containers.dev/implementors/json_reference/)), but for GitPod.

Well, can we not use it `postCreateCommand`? We only really need to set this once. It can just be added to the `postCreateCommand.`

**Comment by MiguelRodo** [comment]
Done
